### PR TITLE
Update installing-solidity.rst

### PR DESCRIPTION
**Pull request by VoR0220**

### CODE REVIEW

File: docs/installing-solidity.rst [hunk 1]
```diff
@@ -115,15 +115,14 @@ they relate only to Alethzero and Mix
...
     cd webthree-umbrella
     ./webthree-helpers/scripts/ethupdate.sh --no-push --simple-pull --project solidity # update Solidity repo
-    ./webthree-helpers/scripts/ethbuild.sh --no-git --project solidity --all --cores 4 # build Solidity
+    ./webthree-helpers/scripts/ethbuild.sh --no-git --project solidity --cores 4 -DEVMJIT=0 # build Solidity
 
 If you opted to install Alethzero and Mix:
 
 .. code-block:: bash
 
-    git clone --recursive https://github.com/ethereum/webthree-umbrella.git
```
Comment (chriseth) [inline docs/installing-solidity.rst:124]: It think we should still mention the clone here, exactly as above.

File: docs/installing-solidity.rst [hunk 2]
```diff
@@ -115,15 +115,14 @@ they relate only to Alethzero and Mix
...
-    cd webthree-umbrella && mkdir -p build && cd build
-    cmake ..
+    ./webthree-helpers/scripts/ethupdate.sh --no-push --simple-pull --project all # update Solidity repo
```
Comment (chriseth) [inline docs/installing-solidity.rst:126]: Does the comment still apply?

File: docs/installing-solidity.rst (hunk 1 above)
Comment (VoR0220) [inline docs/installing-solidity.rst:124]: that's fair. We can do that.

File: docs/installing-solidity.rst (hunk 2 above)
Comment (VoR0220) [inline docs/installing-solidity.rst:126]: I haven't tried the cmake route...either way we need to specify that for OS X that they need to make sure that they set DEVMJIT to 0.

**Comment by chriseth** [comment]
It seems there were conflicts, could you please rebase?

**Pull request closed without merging**